.github/workflows/docker.yml

```diff
@@ -38,9 +44,15 @@ jobs:
           password: ${{ secrets.DOCKERHUB_TOKEN }}
       
       - name: Build and push
-        id: docker_build
```

Comment:
This indeed was not necessary, thanks for cleaning it up!

Reply:
Oh man, I didn't even notice I removed it! lol